params/config_extra.go

```diff
@@ -260,6 +263,12 @@ func (c *ChainConfig) SetNetworkUpgradeDefaults() {
 	if c.MuirGlacierBlock == nil {
 		c.MuirGlacierBlock = big.NewInt(0)
 	}
+	if c.BerlinBlock == nil {
+		c.BerlinBlock = big.NewInt(0)
+	}
+	if c.LondonBlock == nil {
+		c.LondonBlock = big.NewInt(0)
+	}
```

Comment:
basically, with this we will not support subnet-evm networks running without the subnet-evm fork enabled at genesis.
those networks (if they exist) are welcome to directly specify (by modifying `c`) the blocks at which the forks should be activated, which is also likely the path to go in coreth.